### PR TITLE
Prediction fixes

### LINKED ISSUE #116: small DICOM test images no longer managed in git
As of #95, DICOM images in `tests/assets/test_image_data/small` were moved from `git-lfs` directly to `git`. They now appear to be included in `git-lfs`.

## Expected Behavior

DICOM images in `tests/assets/test_image_data/small` should be directly managed `git`

## Current Behavior

DICOM images in `tests/assets/test_image_data/small` are linked via `git-lfs`

## Possible Solution

This appears to be a bug introduced by commit [#740af48](https://github.com/concept-to-clinic/concept-to-clinic/commit/740af48fc8fc248fa47242f3e0ea39db081cb71b) from #76.  I suspect the culprit is [somewhere here in `.gitattributes`](https://github.com/concept-to-clinic/concept-to-clinic/pull/76/files#diff-fc723d30b02a4cca7a534518111c1a66).

## Steps to Reproduce

```bash
git checkout 9b1c31
cd tests/assets/test_image_data/small/LIDC-IDRI-0001/
file  '1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192/-100.000000.dcm'
```

`output`
`1.3.6.1.4.1.14519.5.2.1.6279.../-100.000000.dcm: DICOM medical imaging data`

```bash
git checkout 740af4
file  '1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192/-100.000000.dcm'
```
`output`
`1.3.6.1.4.1.14519.5.2.1.6279.../-100.000000.dcm: ASCII text`

I encountered this while trying to load metadata information as of #33 

## Possible Implementation

@Serhiy-Shekhovtsov @vessemer any ideas?

## Checklist before submitting
- [x] I have confirmed this using the officially supported Docker Compose setup using the `local.py` configuration and ensured that I built the containers again and they reflect the most recent version of the project at the `HEAD` commit on the `master` branch 
- [x] I have searched through the other currently open issues and am confident this is not a duplicate of an existing bug
- [x] I provided a **minimal code snippet** or list of steps that reproduces the bug.
- [x] I provided **screenshots** where appropriate
- [x] I filled out all the relevant sections of this template

### PR DESCRIPTION
**Pull request by reubano**
Additional prediction fixes and optimizations 

### Add docker fixes and optimizations

- move less frequently modified torch layer up front
- remove `python` symlink since it was causing a file exists error
- simplify docker-compose layers and add caching

### Reword prediction error messages

Use more grammatically correct message

### Fix git-lfs typo

Use correct test folder name

### Add small DICOM images (fixes #116)

Put small DICOM images back into `git`

## How Has This Been Tested?

pushed to travis

## CLA
- [x] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

**Comment by lamby** [comment]
@reubano 

Not entirely sure what you'd like me to do here. "docker fixes and optimizations" are all in one commit, so I don't really know a) which are optimisations and which are fixes and b) what they are actually doing. Happy to just go ahead and merge if this all works for you however, I'm just stuck with what my "next action" here is!

**Comment by reubano** [comment]
@lamby Just a second set of eyes to make sure I didn't do anything stupid :).

File: tests/assets/test_image_data/small/LUNA-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797.mhd [hunk 1]
```diff
@@ -1,3 +1,13 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:9a693678a48ec451b218e1b8d2381e0a5d5ccb6e0f8f1b8100a913eae7bfe34b
-size 392
+ObjectType = Image
```
Comment (WGierke) [inline tests/assets/test_image_data/small/LUNA-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797.mhd]: How come this file was wrong / needed a fix?

Comment (reubano) [reply tests/assets/test_image_data/small/LUNA-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797.mhd]: Adds `git-lfs` files to `git`... see #116

Comment (WGierke) [reply tests/assets/test_image_data/small/LUNA-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797.mhd]: I see. Thanks! :)

File: compose/prediction/Dockerfile-dev [hunk 2]
```diff
@@ -3,13 +3,13 @@ ENV PYTHONUNBUFFERED 1
 RUN apt-get update && apt-get install -y tcl tk python3.6 python3.6-tk wget python-opencv
 RUN wget https://bootstrap.pypa.io/get-pip.py
 RUN python3.6 get-pip.py
-RUN ln -s /usr/bin/python3.6 /usr/local/bin/python
```
Comment (reubano) [inline compose/prediction/Dockerfile-dev]: This is the docker fix... I kept getting a file exists error

Comment (lamby) [reply compose/prediction/Dockerfile-dev]: Always good to document in the commit/code ;)

**Comment by lamby** [comment]
> @lamby Just a second set of eyes to make sure I didn't do anything stupid :).

Well LGTM and nothing obviously breaks ;)

**Pull request merged**